crates/consensus/src/lib.rs

```diff
@@ -62,6 +62,8 @@ pub use signed::Signed;
 pub mod crypto;
 pub mod error;
 
+pub mod extended;
```

Comment:
How about a re-export? `pub use extended::Extended`

Using a module named the same as the struct makes for a stuttering-like syntax. I think other transaction wrappers are also being re-exported?

Reply:
yes right, done